src/RelayClient.ts

```diff
@@ -53,7 +53,6 @@ import type { EnvelopingTxRequest } from './common/relayTransaction.types';
 import {
   MISSING_CALL_FORWARDER,
   MISSING_REQUEST_FIELD,
-  NOT_RELAYED_TRANSACTION,
```

Comment:
if we are no longer using this error, we should get rid of it.

Reply:
Done.